Framework/DataHandling/src/LoadRawHelper.cpp

```diff
@@ -714,7 +714,7 @@ void LoadRawHelper::runLoadLog(const std::string &fileName, const DataObjects::W
     loadLog->setPropertyValue("Names", extractLogName(*logPath));
 
     // Force loading two column file if it's an ISIS ICPevent log
-    if (boost::algorithm::ends_with(*logPath, "ICPevent.txt")) {
+    if (boost::algorithm::iends_with(*logPath, "ICPevent.txt")) {
```

Comment:
Was this previously not working because it was using the wrong `ends_with`?

Reply:
I have no idea, but I saw that all the other similar instances in the file were using case insensitive, so I changed it to be consistent.